test/CrateTest/PDO/Http/ServerPoolTest.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * @copyright Interactive Solutions
```

Comment:
@macnibblet I think you want to use the Apache licence here

Reply:
Yep Please the Create-Apache licence header (see other files)